x-pack/plugin/profiler/src/main/java/org/elasticsearch/xpack/profiler/GetProfilingResponse.java

```diff
@@ -139,11 +140,6 @@ public void writeTo(StreamOutput out) throws IOException {
         }
     }
 
-    @Override
-    public RestStatus status() {
-        return error != null ? ExceptionsHelper.status(ExceptionsHelper.unwrapCause(error)) : OK;
```

Comment:
Just checking that changing the status to always return `200 OK` is what you want to do. Today there's no way to return a different status alongside a chunked response, but we could add that feature if you would prefer. That said, most APIs will return `200 OK` even if there are errors within the response, or they'll just throw the exception.

Reply:
Discussed this elsewhere and agreed to stick to `200 OK` here.

Reply:
As discussed offline the current behavior is fine for us. When an exception happens on the transport layer we would ultimately have `RestActionListener#onFailure()` send an appropriate response.